test_suite.json

```diff
@@ -0,0 +1,27 @@
+{
+  "testcases": [
+    {
+      "config": {
+        "requirements": {
+          "duts": {
+            "*": {
+              "count": 1, 
+              "application": {
+                "bin": "/Users/ollpuo01/git/mbed-os-icetea-integration/BUILD/tests/K64F/GCC_ARM/./TEST_APPS/device/exampleapp/exampleapp.bin", 
```

Comment:
hard coded path probably should not be here, what about whole file, is it needed ?